core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,65 @@ var AdminPage = function(){
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="statusMessage"]'));
 
+  // The reload functions are used for mobile testing
+  // done via Browserstack. These functions may cause
+  // a problem when used to run tests directly on Travis.
+  if (general.checkForDevMode) {
```

Comment:
Shouldn't the part in parens end with `()` since it's a function call?

Reply:
Ah, yes! Correct. Also, made other necessary changes here. Thanks!